### ISSUE TITLE
Add support for entrypoint and command for service containers

### ISSUE THREAD

**Issue by mkusaka** [rhysd/actionlint#644]
GitHub Actions now supports [entrypoint and command overrides for service containers](https://github.blog/changelog/2026-04-02-github-actions-early-april-2026-updates/).

From the [documentation](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#jobsjob_idservicesservice_identrypoint), this is now valid:

```yaml
jobs:
  test:
    runs-on: ubuntu-latest
    services:
      redis:
        image: redis
        entrypoint: redis-server
        command: --save 60 1
```

But `actionlint` currently reports:

```text
.github/workflows/test.yaml:9:9: unexpected key "entrypoint" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes" [syntax-check]
.github/workflows/test.yaml:10:9: unexpected key "command" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes" [syntax-check]
```

Could support for `jobs.<job_id>.services.<service_id>.entrypoint` and `jobs.<job_id>.services.<service_id>.command` be added?
